### PR TITLE
Issue 9950 -VADCP - Added parameter PD3822 (error_seawater_velocity) …

### PR DESCRIPTION
**Pull request by vipullakhani**
…in stream "vadcp_velocity_beam"

### CODE REVIEW

File: RELEASE_NOTES [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Version 1.0.15 (2017-10-31)
```
Comment (danmergens) [inline RELEASE_NOTES]: add trailing blank line to match rest of file

File: csv/ParameterDictionary.csv [hunk 2]
```diff
@@ -532,7 +532,7 @@ ADCP_PD0,DICT748,ADCP_PD0,adcp_config,"PD7,PD10,PD11,PD12,PD16,PD863,PD611,PD612
...
 ADCP_PD0,DICT751,ADCP_PD0,adcp_bottom_track_config,"PD7,PD10,PD11,PD12,PD16,PD863,PD1160,PD1161,PD1162,PD1163,PD1164,PD1165,PD1166,PD1206",PD7,,Engineering,Bottom Track Configuration Data,
-VADCP,DICT752,VADCP,vadcp_velocity_beam,"PD7,PD10,PD11,PD12,PD16,PD863,PD617,PD621,PD623,PD651,PD664,PD671,PD672,PD673,PD674,PD675,PD676,PD710,PD719,PD720,PD721,PD722,PD724,PD725,PD726,PD727,PD873,PD874,PD875,PD876,PD3811,PD3812,PD3813,PD3814,PD3819,PD3820,PD3827,PD3828,PD2769,PD2770,PD2771,PD2772,PD7983",PD7,,Science,Velocity Data Products,
+VADCP,DICT752,VADCP,vadcp_velocity_beam,"PD7,PD10,PD11,PD12,PD16,PD863,PD617,PD621,PD623,PD651,PD664,PD671,PD672,PD673,PD674,PD675,PD676,PD710,PD719,PD720,PD721,PD722,PD724,PD725,PD726,PD727,PD873,PD874,PD875,PD876,PD3811,PD3812,PD3813,PD3814,PD3819,PD3820,PD3822,PD3827,PD3828,PD2769,PD2770,PD2771,PD2772,PD7983",PD7,,Science,Velocity Data Products,
```
Comment (danmergens) [inline csv/ParameterDictionary.csv]: Note the only reason this works is that PD3822 maps to a function (PFID31) so this won't affect the cassandra definition.

File: preload_database.sql [hunk 3]
```diff
@@ -3223,30 +3223,30 @@ INSERT INTO "parameter" VALUES(2708,'ph_light_measurements',3,3,NULL,65,5,'Array
...
-INSERT INTO "parameter" VALUES(2736,'temperature24',9,1,NULL,90,7,'Seawater Temperature in Spatial Grid 24',NULL,4,NULL,NULL,NULL,'Seawater Temperature in Spatial Grid 24 is temperature measured at 1 of 24 locations along the thermistor string that is positioned in a spatial array. The position of each thermistor is given in the metadata, using <x y z> coordinates in centimeters with an origin at the seafloor beneath thermistor #2. Units: degrees Celsius',NULL,NULL);
+INSERT INTO "parameter" VALUES(2713,'temperature01',9,1,NULL,90,7,'Seawater Temperature in Spatial Grid 01',NULL,4,NULL,NULL,NULL,'Seawater Temperature in Spatial Grid 01 is temperature measured at 1 of 24 locations along the thermistor string that is positioned in a spatial array. The position of each thermistor is given in the metadata, using <x y z> coordinates in centimeters with an origin at the seafloor beneath thermistor #2. Units: degrees Celsius',3,NULL);
```
Comment (danmergens) [inline preload_database.sql]: why did these change from `NULL` to `3`?

Comment (vipullakhani) [reply preload_database.sql]: Looks like someone changed something somewhere but I can't see where it's changed. it change NULL to 3 , without any of my changes when i ran ./load_preload.py

**Comment by danmergens** [comment]
@renegelinas are you suggesting that the `csv/ParameterDictionary.csv` file be updated to match the changes to `preload_database.sql`?

**Comment by danmergens** [comment]
@renegelinas - so are you suggesting updating `RELEASE_NOTES.md`? If so, please make a change request so we don't merge this prematurely.

File: RELEASE_NOTES [hunk 4]
```diff
@@ -1,3 +1,8 @@
+# Version 1.0.16 (2017-12-20)
+
+- Issue #9950 - VADCP  missing expected parameters and failed creation of true upward velocity
+    * Added parameter PD3822 (error_seawater_velocity) in stream "vadcp_velocity_beam
+
```
Comment (renegelinas) [inline RELEASE_NOTES]: Add a comment explaining that previous changes to ParameterDefs.csv, adding "Science Data" to the temperature parameters' dataproducttype column, were not updated in preload_database.sql.  Those changes are being committed in this issue.

**Pull request merged**